crew_integration/main.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+CREW_PREFIX=/usr/local
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+EXTENSION_PREFIX=~/MyFiles/.extension
+PWA_PREFIX=${CREW_LIB_PREFIX}/pwa
+SERVER=${PWA_PREFIX}/server.rb
+SENDER=${PWA_PREFIX}/send.rb
+app_path=${EXTENSION_PREFIX}/apps/${2}
+extension_id=id_id
```

Comment:
Should we use environment variables if they are set?  For example:
```
CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
```

Reply:
Done